jquery.pjax.js

```diff
@@ -118,15 +118,16 @@ function handleClick(event, container, options) {
 function handleSubmit(event, container, options) {
   options = optionsFor(container, options)
 
-  var form = event.currentTarget
+  var form = event.currentTarget;
+  var $form = $(form);
 
   if (form.tagName.toUpperCase() !== 'FORM')
     throw "$.pjax.submit requires a form element"
 
   var defaults = {
     type: form.method.toUpperCase(),
```

Comment:
At [line 127](https://github.com/defunkt/jquery-pjax/blob/master/jquery.pjax.js#L127), you are already using `form.method`.

I'm not sure if the browser is ensuring the equality `form.method == $(form).attr('method')`, it might be a good idea to keep using `form.method` from the browser.

But just let me know if you still want me to change it :)